src/Modules/SharedModule/components/NavBar/NavBar.tsx

```diff
@@ -15,25 +16,22 @@ export default function NavBar() {
     <>
       <nav className="navbar navbar-expand-lg nav-theme shadow fixed-top ">
         <div className="container-fluid">
-          <a className="navbar-brand">
+          <a
+            role="button"
+            className="navbar-brand"
+            onClick={() => {
```

Comment:
Why onClick on an anchor element and then navigate via useNavigate hook?, why not using Link directly